### ISSUE TITLE
2.24.	Outstanding:MeD: Alt text inaccurate (1.1.1) 

### ISSUE THREAD

**Issue by falcochip** [MeD-DMC/Snap-framework-MeD#41]
The alt text of the image is simply “certificate”. Since the image has words the alt text should be “Certificat de GICMT.”

for FR: Also, the alt text is currently in English and should be in French
